### ISSUE TITLE
Web: Button disabled(false) doesn't enable the button

### ISSUE THREAD

**Issue by hfhbd** [JetBrains/compose-jb#636]
```kotlin
Button(attrs = {
  disabled(false)
}) {
  Text("Enabled button") 
}
 ```

Results into `<button disabled="false">Enabled button</button>`, but should be `<button>Enabled button</button>`.
The content of `disabled` tag is always ignored, only its presence is checked.

Workaround
```kotlin
Button(attrs = {
  if(buttonDisabledCondition) {
    disabled() 
  }
}) {
  Text("Enabled button") 
}
 ```

**Comment by eymar** [comment]
We published a new version - `0.0.0-web-dev-12`. It contains the fix.

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6428">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
